sdk/digitaltwins/azure-digitaltwins-core/src/test/java/com/azure/digitaltwins/core/ModelsTest.java

```diff
@@ -0,0 +1,134 @@
+package com.azure.digitaltwins.core;
+
+import com.azure.core.http.HttpClient;
+import com.azure.core.http.policy.HttpLogDetailLevel;
+import com.azure.core.http.policy.HttpLogOptions;
+import com.azure.core.http.rest.PagedIterable;
+import com.azure.core.util.logging.ClientLogger;
+import com.azure.digitaltwins.core.models.ModelData;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.net.HttpURLConnection;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.function.Consumer;
+
+import static com.azure.digitaltwins.core.TestHelper.DISPLAY_NAME_WITH_ARGUMENTS;
+import static com.azure.digitaltwins.core.TestHelper.assertRestException;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+/**
+ * Sync client implementation of the model tests defined in {@link ModelsTestBase}
+ */
+public class ModelsTest extends ModelsTestBase {
+
+    private final ClientLogger logger = new ClientLogger(ModelsTestBase.class);
```

Comment:
How's this logger being used?

Reply:
For when we log statements during our tests. Currently nothing is logged, but this may be useful for later

Reply:
I've added a few log statements to these model tests now